From da1e1ebfbfd9d0e6e2487b9171b5843f68c60f33 Mon Sep 17 00:00:00 2001
From: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
Date: Fri, 19 Aug 2022 15:23:31 +0300
Subject: [PATCH 086/151] drivers: net: lan865x: allow up to 3 devices

VC4 has 3.

TODO: convert to dynamic structure allocation so there is no longer
hardcoded maximum number of devices.

Signed-off-by: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
---
 drivers/net/ethernet/microchip/lan865x/tc6-conf.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/microchip/lan865x/tc6-conf.h b/drivers/net/ethernet/microchip/lan865x/tc6-conf.h
index 6bad73e72d99..f3b242b48a12 100644
--- a/drivers/net/ethernet/microchip/lan865x/tc6-conf.h
+++ b/drivers/net/ethernet/microchip/lan865x/tc6-conf.h
@@ -51,7 +51,7 @@ extern "C" {
  * \brief Set the maximum amount of parallel TC6 instances. 1 means a single MACPHY hardware is attached
  */
 #ifndef TC6_MAX_INSTANCES
-#define TC6_MAX_INSTANCES   (2)
+#define TC6_MAX_INSTANCES   (3)
 #endif
 
 /**
-- 
2.17.1

